### DISCUSSION TITLE
Bug title form with TJvCaptionButton

### DISCUSSION THREAD

**Discussion by RodrigoMagalskiDev** [project-jedi/jvcl#183] [General]
Hello, we are using TJvCaptionButton in our vcl forms, and we have encountered a problem. When including the CaptionButton, it ends up writing the form's caption, leaving it blank. After resizing the form, or switching windows and returning the focus to the window in question, it displays the caption normally.
Attached is a simulation of the problem that is occurring in the latest version of the demo component made available by you.
![caption_button](https://github.com/project-jedi/jvcl/assets/108557225/bb911862-7b1a-4e26-a554-9d50d634b920)

**Comment by ronaldhoek** [comment]
Hmmm, we've been using this in our application (Delphi XE2) for a long time, but have never seen it before (on Windows 10 and earlier).
However - I'm seeing the same behavior too now (running Windows 11 - 23H2)
